tests/llm/fixtures/test_ask_holmes/158_slack_chat_correct_date/toolsets.yaml

```
@@ -0,0 +1,41 @@
+toolsets:
+  internet:
+    enabled: false
+  prometheus/metrics:
+    enabled: true
+  kubernetes/core:
+    enabled: true
+  kubernetes/logs:
+    enabled: true
+  kubernetes/kube-lineage-extras:
+    enabled: false
+  internet:
```

YAMLlint (1.37.1)

[error] duplication of key "internet" in mapping

(key-duplicates)